Mathlib/Algebra/GroupPower/Order.lean

```diff
@@ -405,8 +403,7 @@ section LinearOrderedCommMonoidWithZero
 
 variable [LinearOrderedCommMonoidWithZero M] [NoZeroDivisors M] {a : M} {n : ℕ}
 
-theorem pow_pos_iff (hn : 0 < n) : 0 < a ^ n ↔ 0 < a := by
-  simp_rw [zero_lt_iff, pow_ne_zero_iff hn]
+theorem pow_pos_iff (hn : n ≠ 0) : 0 < a ^ n ↔ 0 < a := by simp_rw [zero_lt_iff, pow_ne_zero_iff hn]
```

Comment:
This is very minor, and I'm not asking you to change it now, but in the future it would be nice if these kinds of changes were avoided because they needlessly increase the diff, and, as far as style is concerned, the community guidelines are ambivalent on this issue.

Reply:
It doesn't increase the diff here given that those lines were already modified.